numpy/distutils/misc_util.py

```diff
@@ -377,9 +377,40 @@ def blue_text(s):
 #########################
 
 def cyg2win32(path):
-    if sys.platform=='cygwin' and path.startswith('/cygdrive'):
-        path = path[10] + ':' + os.path.normcase(path[11:])
-    return path
+    """Convert a path from Cygwin-native to Windows-native.
+
+    Uses the builting cygpath script to do the actual conversion.
+    Falls back to returning the original path if this fails.
+
+    Handles the default "/cygdrive" prefix as well as "/proc/cygdrive"
+    portable prefix and custom cygdrive prefixes such as "/" or "/mnt".
+
+    Parameters
+    ----------
+    path : str
+       The path to convert
+
+    Returns
+    -------
+    converted_path : str
+        The converted path
+
+    Note
+    ----
```

Comment:
```suggestion
    Notes
    -----
```

The Numpydoc name for the section is supposed to be plural.

Reply:
Done